lib/commands/appstore-upload.ts

```diff
@@ -34,6 +34,8 @@ export class PublishIOS implements ICommand {
 			mobileProvisionIdentifier = args[2],
 			codeSignIdentity = args[3],
 			teamID = this.$options.teamId,
+			exportMethod = this.$options.exportMethod ? this.$options.exportMethod : null,
```

Comment:
@vbresults you can run tests like so: `grunt test` in the root folder of nativesript-cli.